starlette/testclient.py

```diff
@@ -426,30 +429,30 @@ def request(  # type: ignore[override]
         method: str,
         url: httpx._types.URLTypes,
         *,
-        content: httpx._types.RequestContent = None,
-        data: httpx._types.RequestData = None,
-        files: httpx._types.RequestFiles = None,
+        content: typing.Optional[httpx._types.RequestContent] = None,
+        data: typing.Optional[_RequestData] = None,
+        files: typing.Optional[httpx._types.RequestFiles] = None,
         json: typing.Any = None,
-        params: httpx._types.QueryParamTypes = None,
-        headers: httpx._types.HeaderTypes = None,
-        cookies: httpx._types.CookieTypes = None,
+        params: typing.Optional[httpx._types.QueryParamTypes] = None,
+        headers: typing.Optional[httpx._types.HeaderTypes] = None,
+        cookies: typing.Optional[httpx._types.CookieTypes] = None,
         auth: typing.Union[
             httpx._types.AuthTypes, httpx._client.UseClientDefault
         ] = httpx._client.USE_CLIENT_DEFAULT,
-        follow_redirects: bool = None,
-        allow_redirects: bool = None,
+        follow_redirects: typing.Optional[bool] = None,
+        allow_redirects: typing.Optional[bool] = None,
```

Comment:
Why are those `Optional[bool]`? Is it the same in `httpx`?

Reply:
I think you may be right. These should have been USE_CLIENT_DEFAULT.

Reply:
No, it's fine. I found out why.

Reply:
It's because I had to maintain `allow_redirects` alive. :+1:

Reply:
Ok so no need for a followup PR?

Reply:
nop, everything is cool. :pray: